assets/jss/styles.js

```diff
@@ -156,7 +156,7 @@ const styles = theme => {
   const menuActiveBkColor =
     theme.palette?.sideMenu?.activeBkColor ||
     defaultTheme.palette.sideMenu.activeBkColor;
-  const menuActiveBk = 
+  const menuActiveBkWithGradient = 
```

Comment:
revert this change. If we decide to change activeBk property to contain a color and not a gradient, we need to change in styles the name of the property and where it is used so it is better to leave it 'menuactiveBk'